NumberUtilities.java

```diff
@@ -4,29 +4,62 @@
 public class NumberUtilities {
 
     public static String getRange(int stop) {
-        return null;
+        String res = "";
+        for(int i = 0; i < stop; i++){
+            res += i;
+        }
+        return res;
     }
 
     public static String getRange(int start, int stop) {
-        return null;
+        String res = "";
```

Comment:
violation of [`DRY` principle](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwim7Ny1nbXxAhWZX80KHSH6A-MQFjAAegQIBhAD&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FDon%2527t_repeat_yourself&usg=AOvVaw3ALV8DXava0KllhLYVVUYB).
Please make call to the [template method](https://en.wikipedia.org/wiki/Template_method_pattern).

```java
public static String getRange(int start, int stop) {
    return getRange(start, stop, 1);
}
```